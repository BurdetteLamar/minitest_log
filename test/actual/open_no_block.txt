No block given for MinitestLog#open.