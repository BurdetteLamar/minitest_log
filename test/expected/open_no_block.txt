No block given for MinitestLog#new.